pkg/util/priority/priority.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package priority
+
+import (
+	"context"
+
+	schedulingv1 "k8s.io/api/scheduling/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"sigs.k8s.io/kueue/pkg/constants"
+	"sigs.k8s.io/kueue/pkg/workload"
+)
+
+// Priority returns priority of the given workload.
+func Priority(w workload.Info) int32 {
```

Comment:
why not make this a method of `workload.Info`?

Reply:
No special reason. Just keep it as k/k. 
And I also think it's not suitable as a member function. But I can change the input as `kueue.workload` to make it can be used other place in the future.

Reply:
No special reason. Just keep it as k/k. 
And I also think it's not suitable as a member function. But I can change the input as `kueue.workload` to make it can be used other place in the future.